rep24/main.tex

```diff
@@ -779,24 +781,27 @@ \section{Towards a new badge for artifacts}\label{sec:longevity}
 
 We believe that the current badging system is missing one important aspect of the quality of the artifacts: their \emph{longevity}.
 By \emph{longevity} we mean the time an artifact will be in the same state as the state used by the authors.
-As we have seen in Section\ \ref{sec:sop}, not all the popular tools and methods to share source code, package software environment, or platforms to perform experiments on have the same \emph{longevity} guarantees/quality.
+As we have seen in Section\ \ref{sec:sop}, the popular tools and methods to share source code, package software environment, or platforms to perform experiments on differ in their \emph{longevity} guarantees/quality.
 
-Artifacts with long \emph{longevity} are much more valuable and impactful for future researchers to extend, and deserve to be rewarded and have more visibility.
+Artifacts with long \emph{longevity} are much more valuable and impactful as they allow for future researchers to validate the results and build on former research.
+They deserve to be rewarded and have more visibility.
 Table\ \ref{tab:longevity} proposes a grading framework to evaluate the \emph{longevity} of an artifact.
-The artifact is evaluated on three criteria: sharing of the source code, experimental setup used, and software environment. 
-We propose of each criteria 4 levels going from poor but passable quality (1/4) to better (5/4).
+The artifact is evaluated on three \emph{longevity} criteria: how the source code is shared, the availability of the experimental setup, and the longevity of the software environment itself.
+We propose to grade each aspect on 5 levels going from missing (0/4) to best (4/4).
```

Comment:
not sure about "missing"

Reply:
neither am I. I did not dare write "unsatisfactory" or "inadequate" or "insufficient" as it seemed too harsh.

Reply:
i vote "insufficient"